jobs/v3/api_client/commute_search_sample_test.py

```diff
@@ -12,12 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import pytest
+import re
 
-def test_commute_search_sample(capsys):
-    import commute_search_sample
-    import re
+from gcp_devrel.testing import eventually_consistent
```

Comment:
I believe we're discouraging use of gcp_devrel and instead preferring flaky or retry. Is there an advantage to eventually_consistent?

Reply:
`eventually_consistent` is a thin wrapper around retrying, specialized to eventually consistent stuff. I'm not sticking to use gcp_devrel, but the code is more readable and descriptive. I'm more than happy to maintain that library, so there's no worry of it being not maintained.

If you strongly feel we should use `retrying`, I can do that though.